apps/docker/Dockerfile

```
@@ -1,38 +1,37 @@
-# Librelingo is not compatible with Node 16, using Node 14.18.3 image instead
-FROM node:14.18.3 
+FROM archlinux:base-20220227.0.49015
 
-# Setup Librelingo
+# Setup packages
+RUN pacman -Syu --noconfirm git nodejs-lts-fermium gcc yarn python3 python-pip
+
+# Setup Poetry and LibreLingo Python Packages
+RUN curl -sSL https://install.python-poetry.org | python3 - && \
+  cp /root/.local/bin/poetry /bin && \
+  pip install librelingo-yaml-loader && \
+  pip install librelingo-json-export
+  
+# Git clone LibreLingo repo
 RUN git clone https://github.com/LibreLingo/LibreLingo.git
-WORKDIR /LibreLingo
-RUN yarn set version classic && \
+  
+# Setup LibreLingo 
+RUN cd /LibreLingo && \
+  yarn set version classic && \
   yarn
-
-# Setup python, poetry and python modules to be able to convert courses from YAML to JSON
-RUN wget https://www.python.org/ftp/python/3.9.0/Python-3.9.0.tar.xz && \
-  tar xf Python-3.9.0.tar.xz && \
-  cd ./Python-3.9.0 && \
-  ./configure && \
-  make && \
-  make install && \
-  update-alternatives --install /usr/bin/python python /usr/local/bin/python3.9 10 && \
-  ln -s /usr/share/pyshared/lsb_release.py /usr/local/lib/python3.9/site-packages/lsb_release.py && \
-  pip3 install --upgrade pip && \
-  pip install pipx && \
-  pipx install poetry && \
-  export PATH=$PATH:/root/.local/bin && \
-  poetry install && \
+ 
+# Setup tools to import courses
+RUN cd /LibreLingo && \
+  /root/.local/bin/poetry install && \
```

this is weird, why doesn't this work? 

```suggestion
  poetry install && \
```

---

I couldn't figure a way to add /root/.local/bin/ to the $PATH in a dockerfile but this line was also the help this issue ```cp /root/.local/bin/poetry /bin``` as well. The line I specified there was also for the docker-entrypoint.sh file

---

I found a way to update the $PATH by using "ENV PATH="/root/.local/bin/:${PATH}""